src/components/gltf-model.js

```diff
@@ -26,6 +28,9 @@ module.exports.Component = registerComponent('gltf-model', {
       self.model.animations = gltfModel.animations;
       el.setObject3D('mesh', self.model);
       el.emit('model-loaded', {format: 'gltf', model: self.model});
+    }, undefined /* onProgress */, function gltfFailed () {
+      warn('Failed to glTF-model: ' + src);
```

Comment:
The gltfFailed callback will receive an argument, `error` — IMO the warning and event should use that message, as it's probably more informative than the URL. Suggested:

```js
var message = error ? error.message : 'Failed to load glTF model';
warn(message);
el.emit('model-error', {format: 'gltf', message: message});

```